app/mailers/application_mailer.rb

```diff
@@ -48,6 +46,6 @@ def set_perform_deliveries
   protected
 
   def set_delivery_options
-    self.smtp_settings = Settings::SMTP.settings
```

Comment:
Turns out this is the incorrect way to update SMTP config. In spec, checking `ActionMailer::Base.deliveries.last.deliver_methods` after any email is sent out would return `{}`.